_posts/2018-05-17-self-organization.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Self Organization"
+date: 2018-05-17 12:0:00 +0200
+categories: motivation how-we-learn
+author_name: Ugurcan Sengit
+author_url : /author/ugurcansengit
+author_avatar: ugurcansengit
+read_time : 8
+feature_image: posts/2018-04-20-holidaycheck-apprenticeship/apprenticeship.png
+---
+
+I remember the very first day as an apprentice at [HolidayCheck](https://twitter.com/holidaychecklab). Honestly, it's only four months ago, not so hard to recall. The thing is, it feels like it's been a lifetime already. All the things I've learnt during the apprenticeship... One just can't put a price tag on them. Time management and self-organization made them happen.
```

Comment:
I would change "Honestly, it's only... " to "Honestly it was only four months ago". I believe it's not grammatically wrong either way (not sure though) but my first impression was to think it was in the present tense. This makes it clearer that the intention was to talk in past tense.

Reply:
What do you think about changing "One just can't put a price tag on them" to "they're priceless"? Either way, after an ellipsis, (...) you want to use lowercase.

Reply:
I think some adjectives would make that last sentence more meaningfull. "Great time management and efficient self-organization". Or any other positive adjectives you like.